### PR TITLE
[aes] Update coverage refine files

### PR DESCRIPTION
**Pull request by vogelpi**
This PR updates the coverage refine files for both the unmasked and masked implementation of AES.

### CODE REVIEW

**Review by abdullahvarici — APPROVED** [review]
LGTM. I also ran the test for the unmasked implementation and saw that this PR increased the branch coverage from 87% to 91%.

**Comment by vogelpi** [comment]
Thanks for your review @abdullahvarici !

Please let's not merge this yet. This needs some more rework and I need to discuss some issues related to this with @rasmus-madsen .

**Comment by vogelpi** [comment]
I've now added a some pragmas into the FI interfaces. Previously, we had exclusions for these interfaces in the refine files which doesn't sound right to me and also isn't really stable. Because we're not interested in coverage inside these DV files in the first place. But I am no expert. Thus it would be great if e.g. @matutem who introduced these pragmas a while a go could have a look. Is this the way we intend to use such pragmas?

File: hw/ip/aes/dv/cov/refines/aes_UNR.vRefine [hunk 1]
```diff
@@ -1,134 +1,22 @@
...
-    <rule ccType="inst" ccfFlagsMask="37383508603842-42" domain="icc" entityName="aes/u_aes_core/u_aes_control/&quot;gen_fsm[2]&quot;/gen_fsm_n/u_aes_control_fsm_i/u_aes_control_fsm/u_fi_ctrl_fsm/fi_if/7" entityType="top-expr" excTime="1667303729" file="10" im-checksum="880710475" line="89" name="exclude" reviewer="unknown" text="! uvm_hdl_check_path()" ung="0" user="1" vscope="default"></rule>
-    <rule ccType="inst" ccfFlagsMask="37383508603842-42" domain="icc" entityName="aes/u_aes_core/u_aes_control/&quot;gen_fsm[2]&quot;/gen_fsm_n/u_aes_control_fsm_i/u_aes_control_fsm/u_fi_ctrl_fsm/fi_if/8" entityType="top-expr" excTime="1667303729" file="10" im-checksum="880710475" line="95" name="exclude" reviewer="unknown" text="! read" ung="0" user="1" vscope="default"></rule>
-    <rule ccType="inst" ccfFlagsMask="37383508603842-42" domain="icc" entityName="aes/u_aes_core/u_aes_control/&quot;gen_fsm[2]&quot;/gen_fsm_n/u_aes_control_fsm_i/u_aes_control_fsm/u_fi_ctrl_fsm/fi_if/9" entityType="top-expr" excTime="1667303729" file="10" im-checksum="880710475" line="96" name="exclude" reviewer="unknown" text="! uvm_hdl_force()" ung="0" user="1" vscope="default"></rule>
```
Comment (vogelpi) [inline hw/ip/aes/dv/cov/refines/aes_UNR.vRefine:130]: cg_aes_ctrl_fsm_sample

**Review by andreaskurth — APPROVED** [review]
Changes as described in commit messages and comments LGTM. (Reviewing the actual exclusion changes is infeasible with Git diffs.) Thanks for sorting this out, @vogelpi :+1:

**Comment by vogelpi** [comment]
Thanks @andreaskurth for reviewing and approving this!

**Pull request merged**